docs/rackhd/install_os.rst

```diff
@@ -0,0 +1,128 @@
+OS Installation Workflow Support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RackHD workflow support installing Operating System automatically from remote http repository.
+
+Supported OS Installation Workflows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Supported OSes and their workflows are listed in table, and the listed versions have been verified by RackHD, but not limited to these, this table will be updated when more versions are verified.
+
+============= ==================== ==================================================
+OS            Workflow             Version
+============= ==================== ==================================================
+ESXi          Graph.InstallEsx     5.5/6.0
+RHEL          Graph.InstallRHEL    7.0
+CentOS        Graph.InstallCentOS  6.5/7
+Ubuntu        Graph.InstallUbuntu  trusty(14.04)
+SUSE          Graph.InstallSUSE    openSUSE: leap/42.1, SLES: 11/12
+CoreOS        Graph.InstallCoreOS  899.17.0
+============= ==================== ==================================================
+
+
+OS Installation Workflow APIs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start an OS installation workflow:
+
+.. code-block:: REST
+
+    curl -X POST \
+         -H 'Content-Type: application/json' \
+         -d @params.json \
+         <server>/api/1.1/nodes/<identifier>/workflows?name=Graph.InstallCentOS
+
+
+An example of params.json with minimal parameters for installing CentOS workflow:
+
+.. literalinclude:: samples/install-os-minimal-parameters.json
+   :language: JSON
+
+An example of params.json with full set of parameters for installing CentOS workflow:
+
+.. literalinclude:: samples/install-os-maximal-parameters.json
+   :language: JSON
+
+Check the OS installation workflow is active or inactive:
+
+.. code-block:: REST
+
+    curl <server>/api/1.1/nodes/<identifier>/workflows/active
+
+
+Stop the active OS installation workflow:
+
+.. code-block:: REST
+
+    curl -X DELETE <server>/api/1.1/nodes/<identifier>/workflows/active
+
+
+OS Installation Workflow Payload
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All parameters descriptions of OS installation workflow payload are listed below , they are fit to all supported OSes:
+
+=================== ================ ============ ============================================
+Parameters          Type              Flags       Description
+=================== ================ ============ ============================================
+version             String           **required** The version number of target OS that needs to install. **NOTE**: For Ubuntu, the version should be 'trusty' not '14.04'
+repo                String           **required** The OS repository address, currently only supports HTTP
```

Comment:
Suggest to give more details on repos description as different OS has different requirement on detailed path input to RackHD. Or that we can do some enhance in our repo pre-check.